test/test.dart

```diff
@@ -88,9 +88,9 @@ class $RemappedClass {
 }
 
 String? testName;
-Map<String, BaasApp> baasApps = <String, BaasApp>{};
-
+final baasApps = <String, BaasApp>{};
 final _openRealms = Queue<Realm>();
+late Uri? url;
```

Comment:
why a global var

Reply:
This will be removed and I will take the code from master.